src/Applications/LDAS_App/ldas_setup

```diff
@@ -388,10 +393,11 @@ class LDASsetup:
         else:
             raise Exception(
                 'ldsetup should be run either from ..../install/bin '
```

Comment:
ldas_ setup is confirmed to work from Applications? Nice!

Also, I was wondering before. Is ldsetup the old name?
```suggestion
                'ldas_setup should be run either from ..../install/bin '
```

Reply:
NO. That is a leftover. It cannot be run from Application anymore

Reply:
ok, I will take back out
`                 'or from ..../Applications/LDAS_App' `